packages/xrpl/test/integration/utils.ts

```diff
@@ -299,3 +270,70 @@ export async function getXRPBalance(
   }
   return (await client.request(request)).result.account_data.Balance
 }
+
+/**
+ * Retrieves the close time of the ledger.
+ *
+ * @param client - The client object.
+ * @returns - A promise that resolves to the close time of the ledger.
+ *
+ * @example
+ * const closeTime = await getLedgerCloseTime(client);
+ * console.log(closeTime); // Output: 1626424978
+ */
+export async function getLedgerCloseTime(client: Client): Promise<number> {
+  const CLOSE_TIME: number = (
+    await client.request({
+      command: 'ledger',
+      ledger_index: 'validated',
+    })
+  ).result.ledger.close_time
+
+  return CLOSE_TIME
+}
+
+/**
+ * Waits for the ledger time to reach a specific value and forces ledger progress if necessary.
+ *
+ * @param client - The client object.
+ * @param ledgerTime - The target ledger time.
+ * @param [retries=20] - The number of retries before throwing an error.
+ * @returns - A promise that resolves when the ledger time reaches the target value.
+ *
+ * @example
+ * try {
+ *   await waitForAndForceProgressLedgerTime(client, 1626424978, 10);
+ *   console.log('Ledger time reached.'); // Output: Ledger time reached.
+ * } catch (error) {
+ *   console.error(error);
+ * }
+ */
+export async function waitForAndForceProgressLedgerTime(
```

Comment:
This function is only used in the escrow tests, right?

Reply:
Yes @mvadari